inertial_sense_ros:
  ros__parameters:
    port: "/dev/ttyACM0"
    navigation_dt_ms: 4
    baudrate: 921600
    frame_id: "body"
    stream_DID_INS_1: false
    ins1_period_multiple: 1
    stream_DID_INS_2: false
    ins2_period_multiple: 1
    stream_DID_INS_4: true
    ins4_period_multiple: 1
    stream_odom_ins_ned: false
    odom_ins_ned_period_multiple: 1
    stream_odom_ins_enu: false
    odom_ins_enu_period_multiple: 1
    stream_odom_ins_ecef: true
    odom_ins_ecef_period_multiple: 1
    stream_covariance_data: true
    stream_INL2_states: true
    INL2_states_period_multiple: 1
    stream_IMU: true
    imu_period_multiple: 1
    stream_IMU: true
    imu_period_multiple: 1
    stream_GPS1: true
    gps1_period_multiple: 1
    stream_GPS2: true
    gps2_period_multiple: 1
    stream_GPS1_raw: true
    stream_GPS2_raw: true
    gps_raw_period_multiple: 1
    stream_GPS1_info: true
    stream_GPS2_info: true
    gps_info_period_multiple: 1
    GPS1_type: "F9P"
    GPS1_topic: "gps1"
    GPS2_type: "F9P"
    GPS2_topic: "gps2"
    stream_NavSatFix: true
    NavSatFix_period_multiple: 1
    stream_mag: true
    mag_period_multiple: 1
    stream_baro: false
    baro_period_multiple: 1
    stream_preint_IMU: true
    preint_imu_period_multiple: 1
    stream_diagnostics: true
    diagnostics_period_multiple: 1
    publishTf: false
    enable_log: true
    ioConfig: 39624800
    RTK_server_mount: ""
    RTK_server_username: ""
    RTK_server_password: ""
    RTK_connection_attempt_limit: ""
    RTK_connection_attempt_backoff: 2
    RTK_connectivity_watchdog_enabled: true
    RTK_data_transmission_interruption_limit: 1
    RTK_correction_protocol: "RTCM3"
    RTK_server_IP: "127.0.0.1"
    RTK_server_port: 7777
    RTK_rover: false
    RTK_pos_period_multiple: 1
    RTK_rover_radio_enable: false
    RTK_base_USB: false
    RTK_base_serial: false
    RTK_base_TCP: false
    GNSS_Compass: true
    RTK_cmp_period_multiple: 1
    gpsTimeUserDelay: 0
    declination: 0
    INS_rpy_radians: [0, 0, 0]
    INS_xyz: [0, 0, 0]
    GPS_ant1_xyz: [0.9970516019176796085, 0.127508, -0.67736239999999881]
    GPS_ant2_xyz: [0.00010160191768005, 0.086233, -0.0360124000000001]
    GPS_ref_lla: [0, 0, 0]
